src/coreclr/jit/compiler.cpp

```diff
@@ -8300,6 +8283,49 @@ void Compiler::GetStructTypeOffset(
     GetStructTypeOffset(structDesc, type0, type1, offset0, offset1);
 }
 
+#elif defined(TARGET_RISCV64) || defined(TARGET_LOONGARCH64)
+//------------------------------------------------------------------------
+// GetPassFpStructInRegistersInfo: Gets the information on passing of a struct according to hardware floating-point
+// calling convention.
+//
+// Arguments:
+//      structHandle - type handle
+//
+// Return value:
+//      The passing info
+FpStructInRegistersInfo Compiler::GetPassFpStructInRegistersInfo(CORINFO_CLASS_HANDLE structHandle)
+{
+#ifdef TARGET_RISCV64
+#define getInfoFunc getRiscV64PassFpStructInRegistersInfo
+#else
+#define getInfoFunc getLoongArch64PassFpStructInRegistersInfo
+#endif
+
+    FpStructInRegistersInfo ret = info.compCompHnd->getInfoFunc(structHandle);
+#ifdef DEBUG
+    if (VERBOSE)
```

Comment:
```suggestion
    if (verbose)
```
since this is an instance method on `Compiler` anyway...